components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -762,6 +762,42 @@ def _get_signature_from_request(request):
     return rv
 
 
+def _set_quantization_maps(quant_json, img):
+    """
+    Parses 'quant_maps' query string json,
+    setting the quantization map for the listed channel (if given)
+    """
+    if img is None:
+        return
+
+    try:
+        if isinstance(quant_json, (unicode, str)):
+            quant_json = json.loads(quant_json)
+
+        size_c = len(quant_json)
+        if size_c is None:
+            return
+
+        channels = img.getChannels()
+        if (size_c > len(channels)):
+            size_c = len(channels)
+
+        i = 0
+        for c in range(size_c):
+            family = quant_json[i]['family']
+            if family is None:
+                continue
```

Comment:
Do you want to increment ```i``` here? Otherwise ```quant_json[i]['family']``` will be None in the next iteration too?